cmd/abapEnvironmentPushATCSystemConfig.go

```diff
@@ -225,27 +225,22 @@ func buildATCSystemConfigBatchRequest(confUUID string, atcSystemConfiguartionJso
 	var batchRequestString string
 
 	//splitting json into configuration base and configuration properties & build a batch request for oData - patch config & patch priorities
-	var configBaseJson parsedConfigJsonBase
-	err := json.Unmarshal(atcSystemConfiguartionJsonFile, &configBaseJson)
-	if err != nil {
-		return batchRequestString, err
-	}
+	//first remove expansion to priorities to get only "base" Configuration
+	configBaseJsonBody, err := buildParsedATCSystemConfigBaseJsonBody(confUUID, bytes.NewBuffer(atcSystemConfiguartionJsonFile).String())
```

Comment:
Is it not necessary anymore to return in case of an error? The error seems to be overwritten in line 232